dts/bindings/gpio/bflb,bl-gpio.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2021, Gerson Fernando Budke <nandojve@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Bouffalo Lab GPIO node
+
+compatible: "bflb,bl-gpio"
+
+include: [gpio-controller.yaml, base.yaml]
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: false
+
+    label:
+      required: true
+
+    "#gpio-cells":
+      const: 0
```

Comment:
Are you sure? How can you implement the GPIO API without pin and flags cells here?

Please add some more detailed comments explaining this, at least to the commit log, when taking this out of draft.